tools/build.csx

```diff
@@ -64,129 +66,75 @@ var outputDirectory = Path.GetFullPath("./artifacts/output");
 static var testsDirectory = "./artifacts/tests";
 
 // targets
-var targets = new TargetDictionary();
+Add("default", DependsOn("unit", "integ", "spec", "approve", "pack"));

```

Comment:
I appreciate the brevity, but slightly prefer the explicitness of `Targets.Add` (and `.Run`). It's not a huge deal for me though; if I'm alone in this, I'll go with the flow.

Reply:
Agreed; `Add` is a little too generic for my taste. Otherwise, I love the changes!

Reply:
Thanks for the feedback! I can see why this may look a little odd in the context of this script. Elsewhere, I've moved away from scripting entirely and [I use Bullseye in a regular .NET project named `Targets`](https://github.com/xbehave/xbehave.net/blob/6d798877b4b891ce95ea8f2c5f9bee3d56a26bae/targets/Program.cs). Hence the context of "targets" is already set and I think the plain `Add()` makes more sense. But in the context of a script named `build.csx` it does indeed look a little odd.

What about `DependOn()`? Would you prefer that qualified with the type name or not? If not, then we have to have both of these `using` statements at the top of the file:

```c#
using Bullseye;
using static Bullseye.Targets;
```

I'm also now thinking whether the API design is optimal. Maybe this would make more sense:

```c#
using Bullseye.Targets;

AddTarget(...)
RunTargets(...)
```

Reply:
> I'm also now thinking whether the API design is optimal. Maybe this would make more sense:
>
> ```csharp
> using Bullseye.Targets;
> 
> AddTarget(...)
> RunTargets(...)
> ```

It would be my favorite approach 👍

Reply:
I like `AddTarget` and `RunTargets`. In this case, `DependsOn` would stay as is? I think in context, that makes sense. You'd still have a `using static`, yes?